src/OpenSage.Launcher/Program.cs

```diff
@@ -68,6 +68,7 @@ public static void Main(string[] args)
             }
 
             Platform.Start();
+            LanguageSetting.ReadFromRegistry(definition);
```

Comment:
This only works for the launcher, but the engine can also be run via unit tests and the data viewer.